test/fixtures/skip/default/expected.js

```diff
@@ -0,0 +1,6 @@
+var _div$statics = ["__skip", "true"];
```

Comment:
I use skipped elements as containers for nesting components and being able to tell if a given element is a container by looking up its `skip` attribute is certainly a plus for me.